control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -7,6 +7,7 @@ import (
 	"encoding/base64"
 	"encoding/hex"
 	"fmt"
+	errors2 "github.com/pkg/errors"
```

Comment:
We aren't deliberately using this package anywhere else; you can do error wrapping with the stdlib [as of Go 1.13](https://blog.golang.org/go1.13-errors)

Reply:
good catch I was confused on what the %w was doing that is good to know